lib/libmbedtls/mbedtls/library/cmac.c

```diff
@@ -226,17 +240,11 @@ int mbedtls_cipher_cmac_starts( mbedtls_cipher_context_t *ctx,
             return( MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA );
     }
 
-    /* Allocated and initialise in the cipher context memory for the CMAC
-     * context */
-    cmac_ctx = mbedtls_calloc( 1, sizeof( mbedtls_cmac_context_t ) );
-    if( cmac_ctx == NULL )
-        return( MBEDTLS_ERR_CIPHER_ALLOC_FAILED );
-
-    ctx->cmac_ctx = cmac_ctx;
+    /*check if cmac ctx had been allocated by mbedtls_cipher_cmac_setup()*/
```

Comment:
minor: spaces after `/*` and before `*/`. Maybe also s/check/Check/.
Also fix indentation at lines 244-..247.

Reply:
I'll fix